src/components/Logo.js

```diff
@@ -0,0 +1,16 @@
+require('normalize.css/normalize.css');
+require('styles/App.css');
+
+import React from 'react';
+
+class LogoComponent extends React.Component {
+  render() {
+    let image = require('../images/food.jpg');
```

Comment:
the good practice is to use `const` if using `let` is not necessary - this could be also written as `import image from '../images/food.jpg'` and be moved at the beginning of the file (warning: some additional webpack loader may be needed to support importing image files)